### PR TITLE
Chapter 8

### PR DESCRIPTION
**Pull request by nuriman1**

### CODE REVIEW

**Comment by nuriman1** [comment]
ready

File: app/views/sessions/new.html.erb [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+
+       <%= f.label :password %>
+       <%= f.password_field :password, class: 'form-control' %>
+
+       <%= f.submit "Log in", class: "btn btn-primary" %>
+     <% end %>
+
+     <p>New user? <%= link_to "Sign up now!", signup_path %></p>
+   </div>
+ </div>
```
Comment (saobv-2223) [inline app/views/sessions/new.html.erb]: remove first space

**Comment by nuriman1** [comment]
ready

File: app/views/sessions/new.html.erb [hunk 2]
```diff
@@ -0,0 +1,19 @@
+<% provide :title, "Log in" %>
+<h1>Log in</h1>
+
+<div class="row">
+  <div class="col-md-6 col-md-offset-3">
+    <%= form_for :session, url: login_path do |f| %>
+
+      <%= f.label :email %>
+      <%= f.email_field :email, class: 'form-control' %>
```
Comment (saobv-2223) [inline app/views/sessions/new.html.erb]: using "

File: app/views/sessions/new.html.erb [hunk 3]
```diff
@@ -0,0 +1,19 @@
...
+
+      <%= f.label :password %>
+      <%= f.password_field :password, class: 'form-control' %>
```
Comment (saobv-2223) [inline app/views/sessions/new.html.erb]: using "

File: app/views/sessions/new.html.erb (hunk 1 above)
Comment (nuriman1) [reply app/views/sessions/new.html.erb]: done

File: app/views/sessions/new.html.erb (hunk 2 above)
Comment (nuriman1) [reply app/views/sessions/new.html.erb]: done

File: app/views/sessions/new.html.erb (hunk 3 above)
Comment (nuriman1) [reply app/views/sessions/new.html.erb]: done

**Comment by nuriman1** [comment]
ready

**Review by chungpv-1008 — COMMENTED** [review]
merged

**Pull request merged**